USER_GUIDE.md

```diff
@@ -48,6 +49,25 @@ static class IndexData {
 }
 ```
 
+## Create a client
+
+```java
+Transport transport = new RestClientTransport(restClient, new JacksonJsonpMapper()); 
+OpenSearchClient client = new OpenSearchClient(transport);
+```
+
+The `JacksonJsonpMapper` class (2.x versions) only supports Java 7 objects by default.
+[Java 8 modules](https://github.com/FasterXML/jackson-modules-java8) to support JDK8 classes
+such as the Date and Time API (JSR-310), `Optional`, and more can be used by including
+[the additional datatype dependency](https://github.com/FasterXML/jackson-modules-java8#usage)
+and adding the module.  For example, to include JSR-310 classes:
```

Comment:
> Get rid of the line wrap, we don't need it in GitHub Markdown.

While it is not _needed_, it makes no difference to the final rendered product, much like line length in code editors or "pretty" JSON makes no difference to processing it.  However, there are readability concerns associated with editing the file.

GitLab has [a good summary of the two sides of this debate](https://about.gitlab.com/blog/2016/10/11/wrapping-text/), ultimately concluding in their style guide:

> Split up long lines, this makes it much easier to review and edit. Only double line breaks are shown as a full line break in [GitLab markdown](https://docs.gitlab.com/ee/user/markdown.html). 80-100 characters is a good line length.

The article includes two screencaps showing how this improves readability/editability when dealing with the raw content:

![image](https://user-images.githubusercontent.com/9291703/201163070-26d99b10-7c7b-40de-b9d3-867b460bf48f.png)

![image](https://user-images.githubusercontent.com/9291703/201163119-c8e40e72-f215-46fd-b0e4-eaa6a02a075c.png)

Similarly Google's [Markdown Style Guide](https://google.github.io/styleguide/docguide/style.html#character-line-limit) specifies:

> Obey projects’ character line limit wherever possible. Long URLs and tables are the usual suspects when breaking the rule. (Headings also can’t be wrapped, but we encourage keeping them short). Otherwise, wrap your text:
> 
> ```md
> Lorem ipsum dolor sit amet, nec eius volumus patrioque cu, nec et commodo
> hendrerit, id nobis saperet fuisset ius.
> 
> *   Malorum moderatius vim eu. In vix dico persecuti. Te nam saperet percipitur
>     interesset. See the [foo docs](https://gerrit.googlesource.com/gitiles/+/master/Documentation/markdown.md).
> ```
> Often, inserting a newline before a long link preserves readability while minimizing the overflow:
> ```md
> Lorem ipsum dolor sit amet. See the
> [foo docs](https://gerrit.googlesource.com/gitiles/+/master/Documentation/markdown.md)
> for details.
> ```

I tried to observe the last comment (wrap before long URLs) for precisely this reason.

All that said, I did look at all the other markdown files at this project and while I disagree with the style, I'll change to be consistent. :-)